Airports/EGPH/SMR/Labels.txt

```diff
@@ -74,27 +82,17 @@
 "32" N055.56.48.617 W003.21.07.059 standHold
 "33" N055.56.46.845 W003.21.08.188 standHold
 "34" N055.56.45.834 W003.21.09.197 standHold
-"45" N055.56.43.891 W003.20.55.025 standHold
-"46" N055.56.41.297 W003.20.58.251 standHold
-"47" N055.56.43.174 W003.21.00.636 standHold
-"49" N055.56.42.942 W003.21.01.276 standHold
-"50" N055.56.42.722 W003.21.01.877 standHold
-"51" N055.56.42.454 W003.21.02.565 standHold
-"52" N055.56.39.269 W003.21.03.631 standHold
-"53" N055.56.44.335 W003.21.01.989 standHold
-"54" N055.56.44.074 W003.21.02.697 standHold
-"55" N055.56.43.864 W003.21.03.313 standHold
-"56" N055.56.43.632 W003.21.04.004 standHold
-"57" N055.56.43.402 W003.21.04.632 standHold
-"58" N055.56.43.151 W003.21.05.312 standHold
-"59" N055.56.42.939 W003.21.05.955 standHold
-"60" N055.56.42.701 W003.21.06.604 standHold
+"50" N055.56.41.680 W003.20.11.710 standHold
+"51" N055.56.40.950 W003.20.09.260 standHold
+"52" N055.56.40.190 W003.20.06.830 standHold
+"53" N055.56.39.430 W003.20.04.400 standHold
+"54" N055.56.38.680 W003.20.01.970 standHold
 "100" N055.56.46.203 W003.22.06.053 standHold
 "102" N055.56.47.887 W003.22.06.124 standHold
-"104" N055.56.49.665 W003.22.06.169 standHold
-"105" N055.56.50.528 W003.22.05.538 standHold
-"107" N055.56.51.450 W003.22.05.529 standHold
-"108" N055.56.52.555 W003.22.05.783 standHold
+"103" N055.56.49.290 W003.22.07.630 standHold
+"104" N055.56.50.710 W003.22.08.240 standHold
+"105" N055.56.52.120 W003.22.08.800 standHold
+"106" N055.56.54.160 W003.22.09.320 standHold
 "113" N055.56.50.593 W003.21.15.503 standHold
```

Comment:
Does this stand even exist? 
![default](https://user-images.githubusercontent.com/18355854/29998189-91a9a23a-902d-11e7-9669-5019b6a3049a.png)

Reply:
This should be stand 29. Not sure why it's 113.